hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -553,6 +584,10 @@ module flash_ctrl import flash_ctrl_pkg::*; (
   % endfor
 
   // qualify reg2hw settings with creator / owner privileges
+  logic iso_flash_wr_en;
+  assign iso_flash_wr_en = lc_provision_wr_en[FlashWrLcInfoCfg] == lc_ctrl_pkg::On |
+                           lc_iso_flash_wr_en == lc_ctrl_pkg::On;
```

Comment:
ah, i'm glad you reminded me. I actually need to sync up with you again on `lc_nvm_debug_en`.  Because we might actually need to cut that one off earlier exactly because of the same reason.

This function is meant to protect a secret programmed during wafer stage all the way through FT.  So we don't allow anyone to functionally read it (but keep allowing writes), but the debug enable on the hand might need to be severed earlier to make sure it can't be used to usurp this protection.   But let me think about this a bit...